nmap_dns.cc

```diff
@@ -784,7 +793,10 @@ static void read_evt_handler(nsock_pool nsp, nsock_event evt, void *) {
               }
               output_summary();
               stat_ok++;
+              /*Since we have got the answer we can avoid iterating over all answers*/
```

Comment:
@bonsaiviking but if we use `processing_successful` in that sense then it will terminate the loop for both records while it might be the case that we have both PTR record and CNAME record in response and to avoid system resolver we should iterate over the loop until and unless we encounter PTR record.